AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/CreateKeys.dfy

```diff
@@ -325,12 +330,16 @@ module {:options "/functionSyntax:4" } CreateKeys {
           storage
         );
       case "2" =>
-        // TODO-HV-2-M3: Support Version in HV-2
-        output := Failure(Types.KeyStoreException(
-                            message :=
-                              "Active Branch Key found with hierarchy-version 2.\n"
-                              + "VersionKey operation does not support versioning hierarchy-version 2.\n"
-                          ));
+        output := VersionActiveBranchKeyVersion2(
```

Comment:
```suggestion
        output := VersionActiveBranchKeyVersionHV2(
```

Reply:
I personally like the current name, it's less words and technically doesnt use the not so obvious code name for the project.